includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -765,7 +765,7 @@ protected static function normalize_url_for_storage( $url ) {
 		// Query args to be removed from validated URLs.
 		$removable_query_vars = array_merge(
 			wp_removable_query_args(),
-			[ 'preview_id', 'preview_nonce', 'preview', QueryVar::NOAMP ]
+			[ 'preview_id', 'preview_nonce', 'preview', QueryVar::NOAMP, AMP_Validation_Manager::VALIDATE_QUERY_VAR, 'amp_store' ]
```

Comment:
> Side note, during site scanning we'll be wanting to evaluate the URLs as if Standard mode is enabled. Currently validation is performed based on whatever the currently-selected mode is, which is by default Reader mode currently. So we should come up with a way to temporarily override the mode, perhaps with a query parameter.

In light of the other [comment](https://github.com/ampproject/amp-wp/issues/4719#issuecomment-914629682), we may have to add one more query parameter that would override the template mode to Standard.

Currently, for a fresh plugin install, we run into an error at the Site Scan step (in Onboarding Wizard). We're attempting to validate a page but no template mode seems to be selected yet:

```
code: "AMP_NOT_AVAILABLE"
message: "The requested URL is not an AMP page. AMP may have been disabled for the URL. If so, you can forget the Validated URL."
```

Reply:
Yes, you're right. The URL parameter should (1) force Standard mode and (2) force AMP to be enabled for all templates.

Reply:
I'll add that to this PR.